### DISCUSSION TITLE
Numbering documents

### DISCUSSION THREAD

**Discussion by maxw0206** [david-allison/manx-search-data#699] [General]
Could the document numbers in Github be present on the search screen? Then we could easily identify which doc. we were talking about, without spelling out the title.

**Comment by david-allison** [comment]
For now, the identifier is in the web address: https://manxcorpus.com/docs/yn-emshyr?q=as. `yn-emshyr` is guaranteed to be unique.

I'll move this to the header of the text in https://github.com/david-allison-1/manx-corpus-search/issues/53#issuecomment-875610420
